clock-impl/src/main/java/io/deephaven/clock/impl/JdkInternalsRealTimeClock.java

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright (c) 2016-2022 Deephaven Data Labs and Patent Pending
+ */
+package io.deephaven.clock.impl;
```

Comment:
We'll want to use a different license for clock-impl if we use the code as-is since we are using systemUTC rather directly here. That may not be true if we change our strategy for calling VM